README.md

```diff
@@ -1,8 +1,9 @@
-[![Build Status](https://travis-ci.org/mindscreen/yarnlock.svg?branch=master)](https://travis-ci.org/mindscreen/yarnlock)
-[![Coverage Status](https://coveralls.io/repos/github/mindscreen/yarnlock/badge.svg?branch=master)](https://coveralls.io/github/mindscreen/yarnlock?branch=master)
-
 # Mindscreen\YarnLock
 
+[![CircleCI](https://circleci.com/gh/mindscreen/yarnlock/tree/master.svg?style=svg)](https://circleci.com/gh/mindscreen/yarnlock/?branch=master)
+[![codecov](https://codecov.io/gh/mindscreen/yarnlock/branch/master/graph/badge.svg?token=HSF16OGPyr)](https://app.codecov.io/gh/mindscreen/yarnlock/branch/master)
```

Comment:
Where does the token come from?

Reply:
That hash basically does nothing. I can be any string.